core/fx-jsx.js

```
@@ -0,0 +1,617 @@
+ * @file fx-jsx.js
+ * @description FX JSX Parser with Native Reactivity
+ */
+
+class FXLogger {
+    static log(level, message, data = {}) {
+        console.log(`[FX-JSX:${level.toUpperCase()}]`, message, data);
+    }
+    static error(message, error) { this.log('error', message, { error }); }
+    static warn(message, data) { this.log('warn', message, data); }
+    static info(message, data) { this.log('info', message, data); }
+}
```

Biome (1.9.4)

[error] Avoid classes that contain only static members.

Prefer using simple functions instead of classes with only static members.

(lint/complexity/noStaticOnlyClass)